src/ProcessService/ProcessServiceUtils.cpp

```diff
@@ -191,9 +191,17 @@ std::optional<TotalCpuTime> GetCumulativeTotalCpuTime() {
 
   std::vector<std::string_view> splits = absl::StrSplit(first_line, ' ', absl::SkipWhitespace{});
 
+  auto begin_field = splits.begin() + 1;  // The first field is the cpu ID `cpuXX`, so we skip that.
+  auto end_field = std::min(begin_field + 8, splits.end());
+  // We accumulate up to 8 fields. (Depending on what kernel version is available there might be
+  // fewer or more fields. We skip the 9th (`guest`) and the 10th field (`guest_nice`) because they
+  // are already included in the first field (`usertime`).)
```

Comment:
Also irrelevant: I would drop the outer parentheses since you have inner ones already.
```suggestion
  // We accumulate up to 8 fields. Depending on what kernel version is available there might be
  // fewer or more fields. We skip the 9th (`guest`) and the 10th field (`guest_nice`) because they
  // are already included in the first field (`usertime`).
```

Reply:
Makes sense. Thx. Done.